sentry/sentry.go

```diff
@@ -0,0 +1,82 @@
+// Package sentry implements logic for using Sentry for error reporting.
+package sentry
+
+import (
+	"context"
+	"fmt"
+	"github.com/getsentry/sentry-go"
+	"github.com/golang/glog"
+)
+
+var initialized bool
+
+// Initialize initializes a Sentry connection with the given client option values.
+func Initialize(dsn, environment, release string) error {
+	if initialized {
+		return fmt.Errorf("Sentry Initialize called after initialization")
+	}
+
+	var clientOptions sentry.ClientOptions
+
+	clientOptions.Dsn = dsn
+	clientOptions.Environment = environment
+	clientOptions.Release = release
+
+	if err := sentry.Init(clientOptions); err != nil {
+		return err
+	}
+
+	initialized = true
+
+	return nil
+}
+
+// SetHubOnContext clones the current hub and sets it on the given context.
+func SetHubOnContext(ctx context.Context) context.Context {
+	hub := sentry.CurrentHub().Clone()
+
+	return sentry.SetHubOnContext(ctx, hub)
+}
+
+// getHubFromContext gets the current Sentry hub from the given context. If the context is missing a
+// Sentry hub, this function logs an error and returns nil. If Sentry has not been initialized, this
+// function also returns nil.
+func getHubFromContext(ctx context.Context) *sentry.Hub {
+	if !initialized {
+		glog.V(3).Info("getHubFromContext: Sentry not initialized")
```

Comment:
Let's make this V5. A lot of users run with `--v=3` and I would rather not have this clutter up their logs.